zipkin-receiver-kafka/src/main/scala/com/twitter/zipkin/receiver/kafka/KafkaStreamProcessor.scala

```diff
@@ -28,4 +32,16 @@ case class KafkaStreamProcessor[T](
       }
     }
   }
+
+  def retry[T](n: Int = retrycount)(fn: => T): T = {
+    Try { fn } match {
+      case Success(x) => x
+      case _ if n > 1 => retry(n - 1){
+          Thread.sleep(sleeptime)
+          log.debug("queue full, waiting 1s")
```

Comment:
are we certain that the reason we hit this is because of  a queue being full? why not log the exception?

Something to the effect of:
```
case Failure(e) if n > 1 => retry... { ... log.debug(s"Retrying in 1s: $e") ... }
```